modules/app.rb

```diff
@@ -120,4 +124,42 @@ def list_rentals
       end
     end
   end
+
+  private
+
+  def save
+    File.write('people.json', JSON.generate(@people))
+    File.write('books.json', JSON.generate(@books))
+    File.write('rentals.json', JSON.generate(@rentals))
+  end
+
+  def load_people
+    return [] unless File.file?('people.json')
+
+    JSON.parse(File.read('people.json')).map do |person|
+      if person['json_class'] == 'Student'
+        Student.new(person['age'], person['name'], person['parent_permission'])
+      elsif person['json_class'] == 'Teacher'
+        Teacher.new(person['age'], person['specialization'], name: person['name'])
+      end
+    end
+  end
+
```

Comment:
- [ ] There seems to be a bug in your app. I finished creating a person and exited the app. When I rerun the app, it changes the `id` of the person on the JSON file. Kindly fix that. We need to persist the data of the person across multiple iterations without losing or updating the data.

  ![jfke](https://user-images.githubusercontent.com/81468675/226729051-c5d8054b-ea83-4f94-b4d4-2926d77763c3.jpg)